modules/redis/src/main/scala/scalacache/redis/RedisCluster.scala

```diff
@@ -0,0 +1,68 @@
+package scalacache.redis
+
+import redis.clients.jedis.JedisCluster
+import redis.clients.jedis.exceptions.JedisClusterException
+import scalacache.logging.Logger
+import scalacache.redis.StringEnrichment._
+import scalacache.serialization.Codec
+import scalacache.{AbstractCache, CacheConfig, Mode}
+
+import scala.concurrent.duration.{Duration, _}
+import scala.language.higherKinds
+
+class RedisCluster[V](val jedisCluster: JedisCluster)(implicit val config: CacheConfig, val codec: Codec[V])
```

Comment:
Let's call this class `RedisClusterCache` for consistency with the existing classes.